website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -158,22 +158,95 @@ We recommend keeping `critical` alerts non-dismissible as they are essential to
+
+![Ordering alerts](/assets/components/alert/ordering-alerts-color.png)
+
+Alert types have an implied hierarchy and should be ordered starting from `page` and descending in hierarchy to `compact`. This order may depend on how the Alerts are [contextualized](#contextualizing-alerts)
+
+![Ordering alerts of different types](/assets/components/alert/ordering-alerts-type.png)
+
+### Contextualizing Alerts
+
+`Inline` and `compact` Alerts can have more meaning if they are contextualized within the element that is responsible for the Alert. This can help when it’s necessary to have more than one Alert on the page and is relevant for pages that aggregate content like dashboards.
+
+![Example of multiple Alerts contextualized to different parts of the UI](/assets/components/alert/combining-contextualized-alerts.png)
+
+Contextualization is helpful when displaying a specific or localized message; e.g., within a form to highlight validation errors or problems with form submission.
+
+### Placement
+
+How an Alert is contextualized or placed is dependent on the type.
+
+#### Page
 
 Page alerts are placed between the global header navigation and the breadcrumb, next to the left navigation.
 
 ![Placement of page alert](/assets/components/alert/alert-placement-page.png =600x*)
 
-### Inline
+#### Inline
 
 Inline alerts can be added to a section or component or inline with content.
 
 ![Placement of inline alert](/assets/components/alert/alert-placement-inline.png =600x*)
 
-### Compact
+#### Compact
 
 Compact alerts can be added to a section or component or inline with content.
 
-![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
\ No newline at end of file
+![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
+
+#### Mixing Alert types
+
+We recommend against mixing different types of Alerts in the same context, as this can unintentionally convey that different Alert types have the same hierarchical impact.
+
+!!! Dont
+
+Don’t mix different Alert types in the same context.
+
+![Example of different Alert types in the same context](/assets/components/alert/mixing-alert-types-dont.png)
+!!!
+
+!!! Do
+
+Instead, be more specific with how the Alerts are contextualized depending on what the message applies to.
+
+![Example of different Alert types contextualized](/assets/components/alert/mixing-alert-types-do.png)
+!!!
+
+### Alerts within a Reveal
+
+**As a last resort**, when multiple `inline` Alerts are necessary, display the most important Alert to the user within the page while obfuscating additional Alerts within a [Reveal](/components/reveal).
+
+!!! Warning
+
+This method should be **rare and infrequent**, and should only be used if the number of Alerts of the same type exceeds two. Instead, effort should be put into contextualizing Alerts that matter most to the user and in ways that make them easy to resolve.
+!!!
```

Comment:
I don't think I would expect multiple different Alerts in the same reveal. I'll reword the previous point to say:

> **As a last resort**, when multiple Alerts of the same **type** are necessary, display the most important Alert.

But now that I'm thinking through it, I'm not sure if I would recommend this being used for multiple compact Alerts, because those should likely be used in a more specific context. Might be unlikely that that multiple compact Alerts would be displayed next to each other.